functions/mqhandlers.go

```diff
@@ -44,11 +44,22 @@ func NodeUpdate(client mqtt.Client, msg mqtt.Message) {
 	slog.Info("processing node update for network", "network", network)
 	node := config.GetNode(network)
 	server := config.Servers[node.Server]
-	data, err := decryptMsg(server.Name, msg.Payload())
```

Comment:
we need to support backward compatibility 
if host.Version < 0.30.0{
use older decrypt
}

Reply:
Yes, there is compatibility code there, please review the lines following immediately,  lines 50-54

Reply:
what about the encryption?

Reply:
The encrpt/decrypt changes are only applied to peerUpdate, NodeUpdate, HostUpdate(both changed in server/client side).   For other subscription/publish, it still calls the old encrpt/decrypt  function.